docs/tutorials/dev_add_test_group.md

```diff
@@ -0,0 +1,2538 @@
+        super().__init__(test_group=test_group, name=name,
+                         subdir=subdir)
+
+        self.add_step(
+            InitialState(test_case=self, resolution=resolution))
+
+    def validate(self):
+        """
+        Compare ``temperature``, ``salinity`` and ``layerThickness`` from the
+        initial condition with a baseline if one was provided
+        """
+        super().validate()
+        variables = ['temperature', 'salinity', 'layerThickness']
+        compare_variables(test_case=self, variables=variables,
+                          filename1='initial_state/initial_state.nc')
+
+```
+
+Now, we'll make `Default` descend from `YetAnotherChannelTestCase` and remove
+the redundant pieces.  Here's what's left:
+
+```python
+from polaris.ocean.tests.yet_another_channel import YetAnotherChannelTestCase
+from polaris.ocean.tests.yet_another_channel.forward import Forward
+from polaris.ocean.tests.yet_another_channel.viz import Viz
+from polaris.validate import compare_variables
+
+
+class Default(YetAnotherChannelTestCase):
+    """
+    The default test case for the "yet another channel" test group simply creates
+    the mesh and initial condition, then performs a short forward run on 4
+    cores.
+    """
+
+    def __init__(self, test_group, resolution):
+        """
+        Create the test case
+
+        Parameters
+        ----------
+        test_group : polaris.ocean.tests.yet_another_channel.YetAnotherChannel
+            The test group that this test case belongs to
+
+        resolution : float
+            The resolution of the test case in km
+        """
+        super().__init__(test_group=test_group, resolution=resolution,
+                         name='default')
+
+        self.add_step(
+            Forward(test_case=self, ntasks=4, min_tasks=4, openmp_threads=1,
+                    resolution=resolution))
+
+        self.add_step(
+            Viz(test_case=self))
+
+    def validate(self):
+        """
+        Compare ``temperature``, ``salinity``, ``layerThickness`` and
+        ``normalVelocity`` in the ``forward`` step with a baseline if one was
+        provided.
+        """
+        super().validate()
+        variables = ['temperature', 'salinity', 'layerThickness',
+                     'normalVelocity']
+        compare_variables(test_case=self, variables=variables,
+                          filename1='forward/output.nc')
+
+```
+
+### Adding a parameter study
+
+The `yet_another_channel` test group contains several test cases in addition
+to `default`.  The `restart_test` checks whether running the model for one
+times step, writing out a restart file, loading the model state from the
+restart file, and running for another time step produces the same results as
+running for 2 time steps.  The `decomp_test` and `threads_test` check
+whether the results are the same when the model runs on different numbers of
+cores and threads, respectively.
```

Comment:
Actually, I think we can delete this paragraph.

```suggestion
The typical structure for a parameter study is to explore each parameter choice in a separate step (or steps) within a single test case. In addition to running the model for each of these parameter choices, there is typically a separate step for analyzing the behavior across the parameter set, using the output from each of the forward steps. 

A convergence study is one example of this, where each resolution and time step combination is run and then an analysis step computes the rate of convergence from the results. We won't cover this example here, in part because a resolution study involves both a forward step and an initial condition step for each resolution in order to generate unique meshes for each step. You may refer to [insert reference to cosine bell case here] for these details. Instead, we will explore the `rpe_test` for the `baroclinic_channel` test group, which only requires unique forward runs for different viscosity values. 

```

Formatting needed